samples/tls/monitoring/README.md

```diff
@@ -8,25 +8,48 @@ All components have TLS enabled on ingress. Due to limitations in the
 underlying Helm charts, some components might not have TLS enabled in-cluster.
 See the **Limitations and Known Issues** section below for details.
 
-If you use this sample for HTTPS for ingress, the following secrets must be manually populated in the `monitoring` namespace (or `MON_NS` value) **BEFORE** you run any of the scripts in this repository:
+## Ingress
+
+If you use this sample for HTTPS for ingress, the following secrets must be
+manually populated in the `monitoring` namespace (or `MON_NS` value)
+**BEFORE** you deploy cluster monitoring.
 
 * kubernetes.io/tls secret - `prometheus-ingress-tls-secret`
 * kubernetes.io/tls secret - `alertmanager-ingress-tls-secret`
 * kubernetes.io/tls secret - `grafana-ingress-tls-secret`
 
-Generating these certificates is outside the scope of this example. However, you can use the
-process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq) in SAS Viya Administration documentation and specify the `monitoring` namespace.
+Generating these certificates is outside the scope of this example. However,
+you can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq)
+in SAS Viya Administration documentation and specify the `monitoring` namespace.
+
+## In-Cluster TLS
+
+The monitoring components use the following secrets for the TLS certificates
+service-to-service (east-west) traffic:
+
+* `grafana-tls-secret`
+* `prometheus-tls-secret`
+* `alertmanager-tls-secret`
+
+These secrets can be manually created, but if any
+do not exist, then [cert-manager](https://cert-manager.io/) is required to
+create the missing certificates.
+
```

Comment:
You can manually create these certificates. If any do not exist, you must use [cert-manager](https://cert-manager.io/) to create the missing certificates.

Reply:
"you must use cert-manager" is a bit misleading. The scripts will automatically use cert-manager. The user isn't required to "use" cert-manager.

Reply:
How about "However, if any certificates do not exist, [cert-manager](https://cert-manager.io/) automatically creates the missing certificates."

Reply:
The point we're trying to get across is that if you don't create the secrets/certificates yourself, cert-manager is required to be installed in the cluster. If the user manually creates the secrets, cert-manager is **not** required.

Reply:
OK, maybe this: 

[cert-manager](https://cert-manager.io/) is not required if you manually create these certificates. However, if you do not manually create the certificates, cert-manager must be installed in the cluster in order to create any missing certificates.

Reply:
How about: _"If any of the required certificates do not exist, the deployment process will attempt to use cert-manager to generate the missing certificates.  If the required certificates do not exist and cert-manager is not available, the deployment process will fail."_

Reply:
That sounds fine - I'm assuming that replaces the second sentence and that the sentence about manually creating the certificates remains.

Reply:
Yes, that makes sense to me.  But suggest you wait for @BryanEllington before considering discussion finished.

Reply:
Incorporated Greg's suggestion in the latest commit